### PR TITLE
add landing message #1

### PR DESCRIPTION
**Pull request by Wilhelm674**

### CODE REVIEW

**Comment by sbkillmer** [comment]
This is an example comment. "April is not convinced about the <h1> code."

**Comment by sbkillmer** [comment]
This is an example comment. April is not convinced about the code.

**Comment by Wilhelm674** [comment]
Alex answers April’s comments

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-
+<h1>
```
Comment (sbkillmer) [inline index.html]: This is an example comment. April is not convinced about the code.

**Review by sbkillmer — CHANGES_REQUESTED** [review]
This is an example comment. April is not convinced about the code.

File: index.html (hunk 1 above)
Comment (Wilhelm674) [reply index.html]: Alex answers April’s comments

**Review by sbkillmer — APPROVED** [review]

**Pull request merged**